Formula/couchdb.rb

```diff
@@ -92,3 +110,90 @@ def plist
     assert_equal "Welcome", output["couchdb"]
   end
 end
+
+
+__END__
+--- a/src/couch/rebar.config.script
++++ b/src/couch/rebar.config.script
+@@ -76,7 +76,7 @@ ConfigH = [
+     {"JSSCRIPT_TYPE", "JSObject*"},
+     {"COUCHJS_NAME", "\"" ++ CouchJSName++ "\""},
+     {"PACKAGE", "\"apache-couchdb\""},
+-    {"PACKAGE_BUGREPORT", "\"https://issues.apache.org/jira/browse/COUCHDB\""},
++    {"PACKAGE_BUGREPORT", "\"https://github.com/apache/couchdb/issues\""},
+     {"PACKAGE_NAME", "\"Apache CouchDB\""},
+     {"PACKAGE_STRING", "\"Apache CouchDB " ++ Version ++ "\""},
+     {"PACKAGE_VERSION", "\"" ++ Version ++ "\""}
+@@ -123,38 +123,15 @@ end.
+     {unix, _} when SMVsn == "60" ->
+         {
+             "-DXP_UNIX -I/usr/include/mozjs-60 -I/usr/local/include/mozjs-60 -std=c++14 -Wno-invalid-offsetof",
+-            "-L/usr/local/lib -std=c++14 -lmozjs-60 -lm"
++            "-L/usr/local/lib -std=c++14 -lmozjs-60 -lm -lstdc++"
```

Comment:
This might explain why using `llvm` didn't work, btw. If we want to use `llvm` this should probably be `-lc++`. In fact, it might work with Xcode Clang on Catalina/Big Sur if we just made this `-lc++ -stdlib=libc++`

Reply:
This seems like a template related issue, here we use spidermonky 78, the standard library is not specified in [this case](https://github.com/apache/couchdb/blob/e013e3884317b53bde62c71b0d543ddcd4f0c116/src/couch/rebar.config.script#L145), btw, gcc support c++20 better than clang currently.

Reply:
CC @Bo98 in case you have thoughts on this.